_documentation/saml-azure.md

```diff
@@ -84,6 +83,8 @@ kimai:
                     url: 'https://timetracking.example.com/auth/saml/acs'
                 singleLogoutService:
                     url: 'https://timetracking.example.com/auth/saml/logout'
+            security:
+                requestedAuthnContext: false
```

Comment:
Do you have some documentation on that? I do not have access to a Windows machine for testing.

Reply:
I tried various variation of this setting on my machine. setting it to "true" (the default) gives this error when logging in with edge:
`AADSTS75011: Authentication method 'X509, MultiFactor' by which the user authenticated with the service doesn't match requested authentication method 'Password'. Contact the Kimai application owner.`

unfortunately setting it to an array like suggested here: https://github.com/SAML-Toolkits/php-saml#how-it-works

requestedAuthnContext: 
   - 'urn:oasis:names:tc:SAML:2.0:ac:classes:X509'
   - 'urn:oasis:names:tc:SAML:2.0:ac:classes:Password'

makes it work in edge but then you get a similar error like above when logging in with password... 

see also here:
https://learn.microsoft.com/en-us/troubleshoot/azure/active-directory/error-code-aadsts75011-auth-method-mismatch

Reply:
Super interesting. I had a user with an issue that sounds similar, which I was unable to solve: 
https://github.com/kimai/kimai/issues/3698

I wonder if removing that has any security implications. But it seems, it only configures which authentication methods are allowed. And that is really not up to Kimai to decide, but the job of the SAML application owner.

That is a great finding, thank you @tofuSCHNITZEL 👍

Reply:
yes it will fix the problem in https://github.com/kimai/kimai/issues/3698